src/TextField/TextField.js

```diff
@@ -108,10 +123,14 @@ export default class TextField extends Component {
         )}
         <Input
           className={inputClassName}
-          value={value}
+          component={multiLine ? 'textarea' : 'input'}
```

Comment:
I'm wondering, shouldn't it be the `Input` responsibility to render a `textarea`?